src/ripple/server/impl/Port.cpp

```diff
@@ -205,6 +205,29 @@ parse_Port (ParsedPort& port, Section const& section, std::ostream& log)
         }
     }
 
+    {
+        auto const result = section.find("send_queue_limit");
+        if (result.second)
+        {
+            try
+            {
+                port.ws_queue_limit =
+                    beast::lexicalCastThrow<std::uint16_t>(result.first);
+
+                // Queue must be greater than 0
```

Comment:
Is it worth having 0 as indicating an unlimited queue size for use by private servers?